config/v1/types_console.go

```diff
@@ -26,6 +29,26 @@ type Console struct {
 type ConsoleSpec struct {
 	// +optional
 	Authentication ConsoleAuthentication `json:"authentication"`
+	// customization is used to optionally provide a small set of
+	// customization options to the web console.
+	// +optional
+	Customization ConsoleCustomization `json:"customization,omitempty"`
+}
+
+// ConsoleCustomization defines a list of optional configuration for the console.
+type ConsoleCustomization struct {
+	// customURL contains URL and certificate for replacing the default console URL.
+	// +optional
+	CustomURL `json:customURL,omitempty`
+}
+
+// CustomURL replaces the default console URL under which console will be available.
```

Comment:
We need to make sure we're documenting the default behavior when these values aren't specified.